### PR TITLE
Unifying styles of tiles and modals

### PR DESCRIPTION
**Pull request by dan13ram**
Modals: 

before: 
Screenshot from 2022-10-01 18:41:19

after: 
Screenshot from 2022-10-01 18:58:34

Profile/Dashboard Tiles:

before: 
Screenshot from 2022-10-01 18:43:51

after:
Screenshot from 2022-10-01 18:45:08

### CODE REVIEW

File: packages/web/components/Setup/WizardPane.tsx [hunk 1]
```diff
@@ -96,6 +97,8 @@ export const WizardPane = <T,>({
     setValue(field, existing);
   }, [existing, field, setValue]);
 
+  console.log({ status });
```
Comment (vidvidvid) [inline packages/web/components/Setup/WizardPane.tsx:100]: clean this pls :P

File: packages/web/lib/hooks/useSaveCeramicProfile.ts [hunk 2]
```diff
@@ -73,6 +73,7 @@ export const useSaveCeramicProfile = ({
       values?: HasuraStringProps & HasuraEPObjects;
       images?: HasuraImageSourcedProps;
     }) => {
+      console.log('here');
```
Comment (vidvidvid) [inline packages/web/lib/hooks/useSaveCeramicProfile.ts:76]: and this 👍

**Review by vidvidvid — APPROVED** [review]
beautiful cleanup, just pls clean the 2 console.logs and let's ship this

File: packages/web/lib/hooks/useSaveCeramicProfile.ts (hunk 2 above)
Comment (dan13ram) [reply packages/web/lib/hooks/useSaveCeramicProfile.ts]: fixed in 876939c

File: packages/web/components/Setup/WizardPane.tsx (hunk 1 above)
Comment (dan13ram) [reply packages/web/components/Setup/WizardPane.tsx]: fixed in 876939c

**Pull request merged**